metrics-model-libs/funding/definition/funding.md

```diff
@@ -0,0 +1,49 @@
+# Funding
+
+# Why It Matters
+As a project maintainer, core contributors who aren’t compensated are less likely to keep up code quality.
+
+Funding (are contributors and projects funded appropriately, based on project size, maintainer number etc).
+
+More organizational diversity, the community is more robust by the strength of ecosystem partners.
+ 
+
+# User Stories
+
+
+# Metrics in the Metrics Model 
+* [Contribution Attribution](https://chaoss.community/metric-contribution-attribution/) 
+    * What is the ratio of volunteer work, sponsored work, and blended work?
+    * How many contributions are sponsored?
+    * Who is sponsoring the contributions?
+    * What types of contributions are sponsored?
+* [Organizational Influence](https://chaoss.community/metric-organizational-influence/) 
+* [Types of contributions](https://chaoss.community/metric-types-of-contributions/) 
+    * By this metric, we could see the number of organizations of different type of contributions.
+        
+        The number of organizations of different type of Issue/Pull Request/Git commit contributions
+
+* [Labor Investment](https://chaoss.community/metric-labor-investment/) 
+* [Organizational Diversity](https://chaoss.community/metric-organizational-diversity/) 
+    * Organizational diversity expresses how many different organizations are involved in a project and how involved different organizations are compared to one another.
+
```

Comment:
I *might* also include https://chaoss.community/metric-job-opportunities/ as its a trailing indicator of sponsorship.

Reply:
i.e.: If there are jobs, then companies are hiring people who know this project.